src/dns.mli

```diff
@@ -791,6 +791,35 @@ module Edns : sig
       buffer. *)
 end
 
+(** Loc
+
+    A locator record (LOC) is used to express location information associated with domain. *)
+module Loc : sig
+  type t = {
+    lat : int32;
+    long : int32;
+    alt : int64;
+    size : int;
+    horiz_pre : int;
+    vert_pre : int;
+  }
+  (** The type of a Loc record. *)
+
+  val parse : ((int32 * int32 * float) * bool)
+    -> ((int32 * int32 * float) * bool)
+    -> float
+    -> (float * float * float)
+    -> t
```

Comment:
I've no clue what the arguments for this function are. Lots of floating point values.